### ISSUE TITLE
Unexpected white space in second screen

### ISSUE THREAD

**Issue by zhyiyu** [Mortennn/Dozer#58]
First of all I would like to thank you for making this great app!

I have an extra screen and if I am working in one of them, there will be unexpected blank space introduced, please see

![current](https://user-images.githubusercontent.com/26550166/52660495-c896a500-2ecd-11e9-8cf0-51cba19525d6.png)
![other](https://user-images.githubusercontent.com/26550166/52660509-ce8c8600-2ecd-11e9-9878-4e73e021ebae.png)

Above is the screen I am currently working with (screen _a_), below is the other screen (screen _b_). You could see the blank space between the most left two icons.

If I switch to screen _b_, it will appear normally, but screen _a_ would have the same blank space that screen _b_ previously have.

Thanks again for making this wonderful app!

**Comment by ankyliu** [comment]
Maybe it's not the Dozer, it's the App itself.

**Comment by zhyiyu** [comment]
@ankyliu 

You are correct! It is _dropbox_ that is causing the problem, I moved it in between _google drive_ and _mac id_, the extra space is now appearing between _dropbox_ and _mac id_!

**Comment by zhyiyu** [comment]
This issue is resolved after I am updated to mac os mojave.

**Comment by zhyiyu** [comment]
> This issue is resolved after I am updated to mac os mojave.

The issue came back...

**Comment by fuzzy76** [comment]
I am seeing this as well. I am not running Dropbox.

**Comment by zhyiyu** [comment]
I think the white space is gone in macOS Catalina.
